_posts/2023-03-06-cockpit-287.md

```diff
@@ -0,0 +1,110 @@
+
+![screenshot of group improvements]({{ site.baseurl }}/images/287-accounts-group-improvements-2.png)
+
+## Storage: The "nofail" and "_netdev" options are exposed as a dropdown
+
+State: **OPEN** (will not be included) @ <https://github.com/cockpit-project/cockpit/pull/17888>
+{:.warn}
+
+New filesystems now default to "nofail", so that they wont prevent you from fixing things up with Cockpit again when they fail to mount on the next boot.
+
+![screenshot of the "nofail" and "_netdev" options are exposed as a dropdown]({{ site.baseurl }}/images/287-storage-the-nofail-and-netdev-options-are-exposed-as-a-dropdown.png)
+
+## If storage file fails to get deleted, the disk will still get detached from the VM, but we show a warning:
+![Screenshot from 2022-11-14 18-48-54]({{ site.baseurl }}/images/287-if-storage-file-fails-to-get-deleted-the-disk-will-still-get-detached-from-the-vm-but-we-show-a-warning-.png)
+
+## Development: Cockpit now supports the esbuild bundler
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit-podman/pull/1210>
+{:.note}
+
+[esbuild](https://esbuild.github.io/) is a much faster bundler alternative for [webpack](https://webpack.js.org/). It also  natively understands [JSX](https://reactjs.org/docs/introducing-jsx.html), so that it is not necessary any more to use [babel](https://babeljs.io/). Cockpit's shared components and bundler plugins in [pkg/lib/](https://github.com/cockpit-project/cockpit/tree/main/pkg/lib) now support both eslint and webpack, so that projects can pick either. [cockpit-podman](https://github.com/cockpit-project/cockpit-podman) moved to esbuild in [this commit](https://github.com/cockpit-project/cockpit-podman/commit/d48f58297f66d2f6390aa85b54883d9be980850b). You may choose to do the same for your project.
+
+
+## Try it out
+
+Cockpit 287, cockpit-machines 285, and cockpit-podman 64 are available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/287)
+* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/?releases=F37&packages=cockpit)
+* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit)
+* [cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/285)
+* [cockpit-machines Fedora 37](https://bodhi.fedoraproject.org/updates/?releases=F37&packages=cockpit-machines)
+* [cockpit-machines Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit-machines)
+* [cockpit-podman Source Tarball](https://github.com/cockpit-project/cockpit-podman/releases/tag/64)
+* [cockpit-podman Fedora 37](https://bodhi.fedoraproject.org/updates/?releases=F37&packages=cockpit-podman)
+* [cockpit-podman Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit-podman)
```

Comment:
Bodhi is now ready:

```
* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/287)
* [Cockpit Fedora 38](https://bodhi.fedoraproject.org/updates/FEDORA-2023-6987aaacad)
* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-2e47853ed4)
* [cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/285)
* [cockpit-machines Fedora 38](https://bodhi.fedoraproject.org/updates/FEDORA-2023-99b4679f77)
* [cockpit-machines Fedora 37]https://bodhi.fedoraproject.org/updates/FEDORA-2023-9893be2f68()
* [cockpit-podman Source Tarball](https://github.com/cockpit-project/cockpit-podman/releases/tag/64)
* [cockpit-podman Fedora 38](https://bodhi.fedoraproject.org/updates/FEDORA-2023-82b8596f0f)
* [cockpit-podman Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-ff01f5ff9e)
```

Reply:
But the release notes aren't? They're still preview mode. We shouldn't be editing them until they're regenerated in normal mode.

Reply:
I can regenerate after lunch and replace the block manually though. Thanks!